app/views/publishers/organisations/schools/preview.html.slim

```diff
@@ -1,4 +1,4 @@
-- content_for :page_title_prefix, t(".page_title", school_group_name: @organisation.trust.name)
+- content_for :page_title_prefix, t(".page_title", school_group_name: @organisation.name)
```

Comment:
Hmm, I wonder here... @organisation will be the trust (look at `app/controllers/publishers/organisations_controller.rb`), rather than the school who's profile you're previewing, right? 

Probably best to have the page title include the name of the school rather than the trust, no?

Perhaps we need to grab the school here: `app/controllers/publishers/organisations/schools_controller.rb` and pass the school to the view and use it instead?

Reply:
@josephhull676 I think we use the very similarly named `app/views/publishers/organisations/preview.html.slim` for individual school profile previews, and the template I've changed in this PR is just for school groups.

Reply:
Ahhh yeah sorry I'm thinking about the organisation school's index page. In that case, you'd want the trust's name in the title. Cool, approved!